docs/guides/vault/edit-vault-data.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_label: 'Edit Vault Data'
+sidebar_position: 2
+---
+
+# Edit Vault Data
+
+This guide will teach you how to set data to an **[LSP9Vault](../../standards/smart-contracts/lsp9-vault.md)** contract. Any data can be attached to the vault and since it supports **[LSP1-UniversalReceiver](../../standards/generic-standards/lsp1-universal-receiver.md)** standard, we will be setting the [**Universal Receiver Delegate**](../../standards/smart-contracts/lsp1-universal-receiver-delegate-vault.md) address inside the storage.
+
+## Setting Data (Universal Receiver Delegate)
+
+The default implementation of the **Universal Receiver Delegate** of the Vault, that we are going to deploy will register the assets received to the storage and and will remove them when their balance equals 0.
+
+### Deploying Universal Receiver Delegate (URD)
+
+:::info
+
+The **Universal Profile** and the **Vault** don't use the same implementation of the Universal Receiver Delegate.
+:::
+
+```typescript title="Deploying the universal receiver delegate of the vault"
+import LSP1UniversalReceiverDelegateVault from '@lukso/lsp-smart-contracts/artifacts/LSP1UniversalReceiverDelegateVault.json';
+
+// create an instance
+let myURDVault = new web3.eth.Contract(LSP1UniversalReceiverDelegateVault.abi, {
+  gas: 5_000_000,
+  gasPrice: '1000000000',
+});
+
+// deploy the universal receiver delegate Vault contract
+await myURDVault
+  .deploy({
+    data: LSP1UniversalReceiverDelegateVault.bytecode
+  })
+  .send({
+    from: myEOA.address,
+  });
+```
+
+### Setting the URD address in the storage
+
+The owner of the Vault could be an **EOA**, or any **other contract**. In our case we will suppose that the owner of the Vault is a Universal Profile that is controlled by a Key Manager.
```

Comment:
maybe you can add a link on "Universal Profile" to: `/standards/universal-profile/introduction`